test/test_hkdf.c

```diff
@@ -22,6 +22,8 @@
 
 #ifdef WP_HAVE_HKDF
 
+static int test_hkdf_fail(void);
```

Comment:
Put void into function declaration below.

Reply:
I can't remove this void, it gives me the following error:
```
test/test_hkdf.c:25:1: error: function declaration isn’t a prototype [-Werror=strict-prototypes]
   25 | static int test_hkdf_fail();
      | ^~~~~~
```
I am able to compile if I have a void both in the prototype and function.  Would that work instead?

Reply:
The compiler is complaining that the function prototype doesn't have void in the parameters.
If you put into the declaration then it sees it as a prototype.
Remove line and put void in to declaration below.